provision/overlay/bin/audioctl

```diff
@@ -82,7 +80,7 @@ case "$VERB" in
 	down)
 		case "$VOLUME_MANAGER" in
 			amixer)
-				amixer -q set Master 2dB-
+				amixer -q set Master 2-
```

Comment:
I had been meaning to rewrite these to use +- 2%. i am unclear on the behavior here - does `2-` imply a decrease by 2% of maximum volume?

Reply:
This decreases audio by 2dB. The previous command threw an error: 
```zsh
% amixer set Master 2dB- 
amixer: Invalid command!
```

Reply:
It worked as written on Ubuntu 16.04 with `amixer version 1.1.0`. However the `2-` variant appears to also work, and is thus more portable.